src/main_app/utils/wikitext/__init__.py

```
@@ -0,0 +1,24 @@
+    "insert_before_methods",
+    "add_other_versions",
+    "update_original_file_text",
+    "create_cropped_file_text",
+    "update_template_page_file_reference",
+    "appendImageExtractedTemplate",
+    "update_original_file_text",
+    "create_cropped_file_text",
+    "get_titles",
+    "get_titles_from_wikilinks",
+    "get_files_list",
+]
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)